utils/test_harness/src/test_harness_cmdqueue.cpp

```diff
@@ -14,6 +14,27 @@ namespace lzt = level_zero_tests;
 
 namespace level_zero_tests {
 
+uint32_t getComputeQueueGroupOrdinal(ze_device_handle_t device) {
```

Comment:
This can be extended to return a vector of all matching ordinals, the user could just use `lzt::getComputeQueueGroupOrdinals(device_handle)[0]` if they only need one (note the 's').

Reply:
Done!